proposals/0257-eliminate-vote-fees.md

```diff
@@ -0,0 +1,362 @@
+
+In order to implement this change, the vote program will track credits $c_v$ as
+stake weighted totals.
+
+### Vote transaction CU accounting
+
+Vote transactions will no longer count towards the global CU limit in the block,
+they will have their own limit of one vote per validator per block. With SIMD-0218
+IVC, there is no longer any benefit to including multiple old votes within a single
+block, instead only the latest vote will be used.
+
+### Validator set size
+
+By removing the fees for votes, we greatly reduce the barrier for entry as a voter.
+This allows attackers to spin up numerous minimally staked nodes and spam votes
+in an effort to slow down or DoS the cluster.
+
+In order to address this we choose to limit the number of validators that can vote
+at any given time.
+
+These validators that are chosen to vote are the top 2000 validators by stake,
+rotating every epoch.
+
+We must abide by the maximum of 18% of the total cluster stake movement allowed
+per epoch,
+Priority will be given to changes in stake on validators that are already within
+the top 2000 validators by stake before adjustments from inactive validators
+becoming active or active validators becoming inactive.
+
+### TVC window
+
+On the 3rd slot in each validator's sequence of 4 blocks, the timely vote credit
+grace period will only last 1 block. Rewards for landing this vote in the second
+slot will be reduced by 3/8 and follow the normal TVC accounting afterwards.
+
+The reasoning behind this change to avoid scenarios where upcoming leaders bribe
+the voter to delay this vote by 1 block in order for the leader to earn more inclusion
+credits.
+
+## New Terminology
+
+1. $V$ is the set of all validators
+2. $v \in V$ is a validator
+3. $s_v \in [0,1]$ is the stake fraction of validator $v$
+4. $c_v \in [0,1]$ is the stake weighted vote credits of validator $v$
+5. $S = \sum\limits_{v \in V} s_v$ is the total cluster stake
+6. $C = \sum\limits_{v \in V} c_v$ is the total cluster stake weighted vote credits
+7. $I$ is the total inflation rewards issued in an epoch
+8. $T$ represents the total supply of SOL normalized to staked supply of sol
+   $S =1$ so if half of the SOL is staked then $T= 2$
+9. $u_v$ are vote inclusion credits for each validator
+
+## Proofs that the proposed system satisfies the desiderata:
+
+It's clear that maximum validator set size satisfies properties 6 and 7 by
+definition. And that capping entry and exit to 18% satisfies 5, also by
+definition. 1. is satisfied by the removal of vote costs and is the point of the
+proposal. Properties 2,3, and 4 are incentive compatibility properties where we
+must spend our time.
+
+First we should verify that, upon receiving a vote, the leader should include it
+in the next block. Each leader $i$ has reward function.
+
+$$ R_i(s_1,\dots,s_n,c_1,\dots,c_n, u_1,\dots,u_n) = I s_i \frac{c_i}{M} + Iu_i -
+  [\sum_{j=1}^n I s_j \frac{c_j}{M} +  u_i]\cdot \frac{s_i}{T} $$
```

Comment:
In the last term, you are adding Sols and u_i (which is either a fraction or credits, not entirely clear). But in any case, this does not make sense.